### PR TITLE
COMMERCE-12431 Feature flag description

### PR DESCRIPTION
**Pull request by smottal**
Sent as follow up of https://github.com/brianchandotcom/liferay-portal/pull/143911

### CODE REVIEW

**Comment by smottal** [comment]
ci:test:sf

File: modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -7179,6 +7179,7 @@ feature.flag.COMMERCE-11287.description=Sell and manage SKUs by adding different
 feature.flag.COMMERCE-11287.title=Units of Measure
 feature.flag.COMMERCE-11922.description=Use the platform's configurable purchasing rules to dictate which product options need to be purchased together when setting up products with options. You can set which product options must be purchased together and/or which product options cannot be combined together to create a valid product bundle. You can also limit the number of products from a particular group of products that can be purchased together.
 feature.flag.COMMERCE-11922.title=Configurable Purchasing Rules
+feature.flag.COMMERCE-12192.description=This setting enables new features on the organization chart widget to allow you to search, view, and edit users, organizations, and accounts.
```
Comment (andrea-ale-sbarra) [inline modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties:7182]: of user

Comment (andrea-ale-sbarra) [reply modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties:7182]:
```suggestion
feature.flag.COMMERCE-12192.description=This setting enables new features on the organization chart widget to allow you to search, view, and edit of users, organizations, and accounts.
```

**Review by andrea-ale-sbarra — CHANGES_REQUESTED** [review]
not suer if we need also the :

**Comment by smottal** [comment]
ci:test:sf

**Comment by smottal** [comment]
Sent directly [here](https://github.com/brianchandotcom/liferay-portal/pull/143961)

**Pull request closed without merging**